less/home-page.less

```diff
@@ -1 +1,155 @@
-// Home Content
\ No newline at end of file
+// Home Content
+.container {
+    width: 800px;
+    margin: auto;
+}
+
+.middle h1 {
+    margin-left: @lft-mrg;
+    margin-bottom: 1%;
+}
+.middle p {
+    margin-left: @lft-mrg;
+    margin-bottom: 2%;
+}
+
+#picture-row {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-top: 1%;
+
+    img {
+        width: 250px;
+        height: 200px;
+        border-radius: 22px;
+        margin-bottom: 3%;
+
+        h1 {
+            margin-bottom: 1%;
+            padding: 1%;
+        }
+    }
+}
+
+.last-walk {
+    margin-top: 10%;
+    display: flex;
+
+        .btn {
+            border: 2px solid @bg-color;
+            text-align: center;
+            width: 22%;
+            background-color: @bg-color;
+            color: white;
+            border-radius: 6px;
+            padding: 3px 2px 3px 2px;
+        
+    }
```

Comment:
Look into using an extension like `Beautify` or `Prettier` to clean up your CSS.